pkgs/tools/security/echidna/default.nix

```diff
@@ -1,54 +1,68 @@
-{ lib
-, fetchpatch
-, fetchFromGitHub
-# Haskell deps
-, mkDerivation, aeson, ansi-terminal, base, base16-bytestring, binary, brick
-, bytestring, cborg, containers, data-dword, data-has, deepseq, directory
-, exceptions, filepath, hashable, hevm, hpack, lens, lens-aeson, megaparsec
-, MonadRandom, mtl, optparse-applicative, process, random, stm, tasty
-, tasty-hunit, tasty-quickcheck, temporary, text, transformers , unix, unliftio
-, unliftio-core, unordered-containers, vector, vector-instances, vty
-, wl-pprint-annotated, word8, yaml , extra, ListLike, semver
+{ haskell
+, haskellPackages
 }:
-mkDerivation rec {
-  pname = "echidna";
-  version = "1.7.3";
+let
+  hevm = haskell.lib.unmarkBroken (
```

Comment:
Shouldn't this be fixed in the `hevm` derivation instead? Might as well have it work in other derivations too.